### ISSUE TITLE
FP: The closure can be replaced with 'strval'

### ISSUE THREAD

**Issue by voku** [kalessil/phpinspectionsea#1367]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 3.0.1 |
| Language level | PHP 7.3                     |

Current behaviour
--
```php
return \array_map(
    static function (int $i): string {
        return (string) self::chr($i);
    },
    \range($start, $end)
);
```

=> inline the closure 

```php
return \array_map(
    'strval',
    \range($start, $end)
);
```

... but this is not correct.

Expected behaviour
--

... maybe something like this:

```php
return \array_map(
    [__CLASS__, 'chr'],
    \range($start, $end)
);
```

Environment details
---
PhpStorm 2019.1.3
Build #PS-191.7479.51, built on June 18, 2019
JRE: 1.8.0_202-release-1483-b53 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.4.0-154-generic

**Comment by voku** [comment]
```[__CLASS__, 'chr']``` behavior is still different, beacuse of missing (string)

**Comment by kalessil** [comment]
Fixed!
